lib/chat_api_web/controllers/registration_controller.ex

```diff
@@ -80,6 +80,34 @@ defmodule ChatApiWeb.RegistrationController do
     conn
   end
 
+  defp send_registration_event(conn) do
+    with %{email: email, account_id: account_id} <- conn.assigns.current_user do
+      %{email: email}
+
+      now = :os.system_time(:seconds)
+
+      param = %{
+        email: email,
+        created_at: now,
+      }
+
+
+      Customerio.identify(account_id, param)
+      |> case do
+        {:ok, _} -> nil
+        {:error , result} -> IO.inspect(result)
```

Comment:
let's also use `Logger.error` here instead of `IO.inspect` so errors will get logged to Sentry 👍 

(we'll need to `require Logger` at the top for this to work)